client/src/components/FileListItem.tsx

```diff
@@ -0,0 +1,70 @@
+import prettyBytes from 'pretty-bytes';
+import { FiX as Remove } from 'react-icons/fi';
+import { FaCircleCheck as Success } from 'react-icons/fa6';
+import { IoMdWarning as Error } from 'react-icons/io';
+
+export interface File {
+  name: string;
+  size: number; // in bytes
+}
+
+const FileListItem = ({
+  file,
+  errorMessage,
+  onRemove,
+  className,
+}: {
+  file: File;
+  errorMessage?: string;
+  onRemove?: () => void;
+  className?: string;
+}) => {
+  const { name, size } = file;
+  const type = name.split('.').slice(-1)[0];
+
+  return (
+    <div
+      className={`${className ?? ''} flex w-full justify-between items-center`}
+    >
+      <div className='flex items-center gap-5'>
+        <img
+          src={`/${type === 'pdf' || type === 'pptx' ? type : 'default'}.svg`}
+          alt=''
```

Comment:
Why is the alt attribute empty? Should we put something like `type` icon (e.g. PDF icon) to make the code more accessible to screen readers or when the image can't be displayed?

Reply:
Empty alt text is used for purely decorative images. These icons aren't providing sighted users with any additional information that a SR user would be missing since the file name already contains the extension that says the file type.